packages/panna-sdk/src/core/README.md

```diff
@@ -63,12 +63,17 @@ await wallet.connect({
 
 ### Utilities & Helpers
 
-6. **[Util](../util/README.md)** - Common utility functions
+6. **[Util](./util/README.md)** - Common utility functions
    - Balance queries and conversions
    - Address validation
    - Token metadata fetching
    - Activity tracking
 
+7. **[Extensions](./extensions/README.md)** - External wallet integration
+   - Connect MetaMask, WalletConnect, and other EIP-1193 wallets
+   - Convert between Panna and external wallet formats
```

Comment:
`Convert between Panna and external wallet formats` sounds quite ambiguous. Can we please rephrase to sound more clear?